stubs/asyncpg/connection.pyi

```diff
@@ -31,7 +31,7 @@ class Connection:
     async def fetchrow(self, query: str, *args: object, timeout: Optional[float] = ...) -> Optional[Record]: ...
     async def copy_from_table(self, table_name: str, output: Any, *, columns: Optional[List[str]] = ..., schema_name: Optional[str] = ..., timeout: Optional[float] = ..., format: Optional[Any] = ..., oids: Optional[Any] = ..., delimiter: Optional[Any] = ..., null: Optional[Any] = ..., header: Optional[Any] = ..., quote: Optional[Any] = ..., escape: Optional[Any] = ..., force_quote: Optional[Any] = ..., encoding: Optional[Any] = ...) -> str: ...
     async def copy_from_query(self, query: str, *args: object, output: Any, timeout: Optional[float] = ..., format: Optional[Any] = ..., oids: Optional[Any] = ..., delimiter: Optional[Any] = ..., null: Optional[Any] = ..., header: Optional[Any] = ..., quote: Optional[Any] = ..., escape: Optional[Any] = ..., force_quote: Optional[Any] = ..., encoding: Optional[Any] = ...) -> str: ...
-    async def copy_to_table(self, table_name: str, source: Any, *, columns: Optional[str] = ..., schema_name: Optional[str] = ..., timeout: Optional[float] = ..., format: Optional[Any] = ..., oids: Optional[Any] = ..., freeze: Optional[Any] = ..., delimiter: Optional[Any] = ..., null: Optional[Any] = ..., header: Optional[Any] = ..., quote: Optional[Any] = ..., escape: Optional[Any] = ..., force_quote: Optional[Any] = ..., force_not_null: Optional[Any] = ..., force_null: Optional[Any] = ..., encoding: Optional[Any] = ...) -> str: ...
+    async def copy_to_table(self, table_name: str, source: Any, *, columns: Optional[list[str]] = ..., schema_name: Optional[str] = ..., timeout: Optional[float] = ..., format: Optional[Any] = ..., oids: Optional[Any] = ..., freeze: Optional[Any] = ..., delimiter: Optional[Any] = ..., null: Optional[Any] = ..., header: Optional[Any] = ..., quote: Optional[Any] = ..., escape: Optional[Any] = ..., force_quote: Optional[Any] = ..., force_not_null: Optional[Any] = ..., force_null: Optional[Any] = ..., encoding: Optional[Any] = ...) -> str: ...
```

Comment:
The file exists to because the asynpg library doesn't add any type annotation to its method. So we define the type stubs here to add type annotation to the methods anyway. This change fixes an error in the stub itself.

Reply:
Oh wow, cool :+1: